invenio_app_ils/ui/src/invenio_app_ils/common/components/SearchBar/SearchBar.js

```diff
@@ -21,11 +21,17 @@ export class SearchBar extends Component {
       executeSearch,
       placeholder,
       queryHelperFields,
+      buttonColor,
+      ...otherProps
     } = this.props;
     return (
       <>
         <Input
-          icon={<Icon onClick={executeSearch} name="search" link />}
+          action={{
+            color: buttonColor ? buttonColor : 'orange',
```

Comment:
I am guessing the colour needs to be overridable because we use the search bar in several places.
- Should the 'orange' be the default one?
- Could use `SearchBar.defaultProps = { buttonColor: 'blue' }` instead of checking it

Reply:
I'm trying to avoid overwriting the primary colour in this way in the themes, and now it is default as orange in the props